frontend/components/commons/forms/questions-form/QuestionsForm.component.vue

```diff
@@ -251,121 +242,103 @@ export default {
         default:
       }
     },
-    onChange({ newOptions, idComponent }) {
+    onChangeTextArea({ newOptions, idComponent }) {
+      this.inputs = this.inputs.map((input) => {
+        if (input.id === idComponent) {
+          input.options = [{ ...newOptions, value: newOptions.text }];
+        }
+        return input;
+      });
+    },
+    onChangeMonoSelection({ newOptions, idComponent }) {
       this.inputs = this.inputs.map((input) => {
         if (input.id === idComponent) {
           input.options = newOptions;
         }
         return input;
       });
     },
+    async sendBackendRequest(responseValues) {
+      try {
+        let responseData = null;
+        if (this.responseId) {
+          responseData = await this.updateRecordResponses(responseValues);
```

Comment:
I think is more clear if you explicit pass the responseId you will update.
```suggestion
          responseData = await this.updateResponseValues(responseId, responseValues);
```